back/src/domains/convention/adapters/PgAssessmentRepository.ts

```diff
@@ -19,8 +19,22 @@ export class PgAssessmentRepository implements AssessmentRepository {
 
     const dto = assessmentSchema.parse({
       conventionId: result.convention_id,
-      establishmentFeedback: result.establishment_feedback,
       status: result.status,
+      ...(result.status === "PARTIALLY_COMPLETED"
+        ? {
+            lastDayOfPresence: result.last_day_of_presence?.toISOString(),
+            numberOfMissedHours: result.number_of_missed_hours,
+          }
+        : {}),
+      endedWithAJob: result.ended_with_a_job,
+      ...(result.ended_with_a_job
+        ? {
+            typeOfContract: result.type_of_contract,
+            contractStartDate: result.contract_start_date?.toISOString(),
+          }
+        : {}),
+      establishmentFeedback: result.establishment_feedback,
+      establishmentAdvices: result.establishment_advices,
```

Comment:
Vu qu'il y a le parsing de schema, autant construire le DTO avec la totale sans présumer du status.

```suggestion
      establishmentFeedback: result.establishment_feedback,
      establishmentAdvices: result.establishment_advices,
      endedWithAJob: result.ended_with_a_job,
      ...result.contract_start_date? {contractStartDate: result.contract_start_date.toISOString()} : {},
      ...result.type_of_contract ? {type_of_contract : result.type_of_contract} : {},
      ...result.last_day_of_presence ? {lastDayOfPresence: result.last_day_of_presence.toISOString()} : {},
      ...result.number_of_missed_hours? {numberOfMissedHours: result.number_of_missed_hours} : {},
```